google/cloud/pubsub/internal/tracing_pull_lease_manager_test.cc

```diff
@@ -83,6 +85,39 @@ TEST(TracingPullLeaseManagerImplTest, AsyncModifyAckDeadlineSuccess) {
                          SpanNamed("test-subscription modack"))));
 }
 
+#if OPENTELEMETRY_VERSION_MAJOR >= 2 || \
+    (OPENTELEMETRY_VERSION_MAJOR == 1 && OPENTELEMETRY_VERSION_MINOR >= 13)
```

Comment:
I wonder if we should move these ifs into `SpanIsRoot()`. And if otel-cpp < 1.13, the matcher would just return true.

Reply:
Yeah... we might want to refactor this in the future